package.json

```diff
@@ -27,6 +27,7 @@
     "test": "wtr \"extension/test/*_test.ts\"",
     "test:small": "wtr --files \"extension/test/background_test.ts\" \"extension/test/data_test.ts\" \"extension/test/docs-annotate-canvas_test.ts\" \"extension/test/rikaicontent_test.ts\"",
     "test:browserstack": "npm run test -- --browserstack",
+    "test:tiny": "wtr --files \"extension/test/background_test.ts\"",
```

Comment:
probably don't need this all the time; If a person wants to run a single test they can use `.only` or just run wtr on the command line right?

Reply:
This one leaked through. Sorry.